packages/shared/lib/services/config.service.ts

```diff
@@ -277,6 +277,31 @@ class ConfigService {
 
         return result.id;
     }
+
+    async copyProviderConfigCreds(
+        fromEnvironmentId: number,
+        toEnvironmentId: number,
+        providerConfigKey: string
+    ): Promise<{ copiedToId: number; copiedFromId: number } | null> {
+        const fromConfig = await this.getProviderConfig(providerConfigKey, fromEnvironmentId);
+
+        if (!fromConfig || !fromConfig.id) {
+            return null;
+        }
+
+        const { id: foundConfigId, ...configWithoutId } = fromConfig;
+        const id = await this.createProviderConfig({
```

Comment:
it's returning * not id

Reply:
True. Updated with https://github.com/NangoHQ/nango/pull/2655/commits/845c6c56b6969b1c75ef14783c6ef7cc5e0de5ab